playbooks/splunk_idxc_deploy.yml

```diff
@@ -21,3 +21,10 @@
     - ../roles/splunk
   vars:
     - deployment_task: configure_idxc_member.yml
+
+- hosts:
+    - search
```

Comment:
You should add `search` [here](https://github.com/splunk/ansible-role-for-splunk/blob/master/playbooks/splunk_idxc_deploy.yml#L5)